_includes/proposals.html

```diff
@@ -2,10 +2,9 @@
   <a id="proposals" tabindex="-1"></a>
   <h2 class="section-headline text-end">State of Proposals</h2>
   <div class="featurelist">
-    <h3>Stage 3</h3>
+    <h3>Current Candidates for the Specification</h3>
     <i class="featurelist__desc">
-      Proposals that have been reviewed, and are being considered as
-      candidates for the spec are considered to be stage 3.
+      This section features proposals that are in Stage 3 of <a href="https://tc39.github.io/process-document/">our process</a>, which means they are close to completion and likely ready to be shipped in engines.
```

Comment:
it was mentioned that "likely ready to be shipped in engines" might be contentions. Is there another way to say this?

Reply:
Stage 3 things are not necessarily likely to be shipped in engines. For example, SIMD.js was Stage 3 for a long time and never shipped in engines.

Reply:
Likely *ready*, which SIMD was - not likely to be shipped.

Reply:
how about we remove the last part of the sentence? "This section features proposals that are in Stage 3 of our process, which means they are close to completion"

Or we can add "and likely ready for consideration as part of the specification of the language" -- its pretty vague.

Reply:
SGTM